### DISCUSSION TITLE
New User - Initial Setup Options Such as Hosting

### DISCUSSION THREAD

**Discussion by imager2019** [inventree/InvenTree#4974] [Q&A] [answered]
Very interested in InvenTree, but I have zero knowledge of Linux-based systems. I am a Windows user, and I see the warnings/cautions that Inventree may not work (at all).
Is there a hosting provider/service that can be recommended to provide a stable environment and the initial setup and configuration service?
Appreciate any feedback.
Thanks. John

**Comment by matmair** [comment, marked as answer]
@imager2019 assuming this is for business this would be an option: https://www.stellarhosted.com/inventree/. You could also use docker on windows (there are official docker/Kubernetes services for windows server).

**Reply by eeintech** [reply]
Is stellarhosted affiliated with InvenTree?

**Reply by SchrodingersGat** [reply]
No, there is no official link between stellarhosted and the InvenTree project team

**Comment by SchrodingersGat** [comment]
@imager2019 are you looking to setup your installation online (cloud) or self hosted?

**Reply by imager2019** [reply]
I have spent the past couple of days looking at options.
I aim to have something I control instead of a hosted Inventory solution (typically very expensive) for supporting a hobby business with about 3,500 items. A service like StellarHosted is an option, but the more I think about it, the more I think I need to set up some type of Linux server to run locally to provide access to Inventree.

Been reading about different distributions, and Debian 12, seems like a possible solution. There would be a steep learning curve for a non-LINUX person. 

If there better/worse solutions for a Linux distribution that is cable of setting up and running Inventree? How would I know if Docker would be compatible with Debian 12, before I spend too much time running down that rabbit-hole??

Thanks..

**Reply by SchrodingersGat** [reply]
@imager2019 the advantage of docker is that it is abstracted from your actual OS. In theory if your OS can run docker, it can run InvenTree!

@matmair is a better authority on the native installer, and platform support there.

**Reply by matmair** [reply]
Debian/Ubuntu is certainly the standard way. If you already have windows server 2019 or newer you could also use the docker engine there.
